game-core/src/main/java/games/strategy/engine/framework/startup/ui/PlayerType.java

```diff
@@ -0,0 +1,97 @@
+package games.strategy.engine.framework.startup.ui;
+
+import java.util.Arrays;
+import java.util.function.Function;
+
+import games.strategy.engine.gamePlayer.IGamePlayer;
+import games.strategy.triplea.TripleAPlayer;
+import games.strategy.triplea.ai.fast.FastAi;
+import games.strategy.triplea.ai.pro.ProAi;
+import games.strategy.triplea.ai.weak.DoesNothingAi;
+import games.strategy.triplea.ai.weak.WeakAi;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * PlayerType indicates what kind of entity is controlling a player, whether an AI, human,
+ * or a remote (network) player.
+ */
+@AllArgsConstructor
+public enum PlayerType {
+  HUMAN_PLAYER("Human", nationName -> new TripleAPlayer(nationName) {
+    @Override
+    public PlayerType getPlayerType() {
+      return PlayerType.HUMAN_PLAYER;
+    }
+  }),
+
+  WEAK_AI("Easy (AI)", WeakAi::new),
+
+  FAST_AI("Fast (AI)", FastAi::new),
+
+  PRO_AI("Hard (AI)", ProAi::new),
+
+  DOES_NOTHING_AI("Does Nothing (AI)", DoesNothingAi::new),
```

Comment:
One thing we might want to consider: On specific maps certain nations can be disabled completely by unchecking a checkbox.
Ideally we'd reuse the AI selection menu for this